compressible/interface_f.f90

```diff
@@ -296,6 +296,8 @@ subroutine riemann_cgf(idir, qx, qy, ng, &
            ut_l    = U_l(i,j,ixmom)/rho_l
         endif
 
+        rhoe_l = U_l(i,j,iener) - 0.5*rho_r*(un_l**2 + ut_l**2)
+
```

Comment:
why is this needed?

Reply:
I noticed when I did the python version that `rhoe_l` is not calculated before that point then is used to calculate `p_l`. The `rho_r` should be a `rho_l` though.

Reply:
huh... indeed.  How did it even work then?